app/models/user.rb

```diff
@@ -27,7 +27,8 @@ class User < ApplicationRecord
   scope :non_admins, -> { where.not('email ~ ?', DFE_EMAIL_PATTERN) }
   scope :active, -> { where.not(accept_terms_date_utc: nil) }
 
-  validates :email, presence: true
+  validates :email, presence: true, format: { with: /@/, message: "must contain @" }
```

Comment:
is that too wide

![image](https://user-images.githubusercontent.com/470137/58336298-5e31c100-7e3b-11e9-89ab-3e43781a7f44.png)

Reply:
Given this is going to be used by internal users at the moment, I don't think we need to be that strict.

Reply:
:cool: